libyara/libyara.c

```diff
@@ -115,6 +145,13 @@ YR_API int yr_finalize(void)
   if (--init_count > 0)
     return ERROR_SUCCESS;
 
+  #ifdef HAVE_LIBCRYPTO
+  int i;
```

Comment:
Just a small note: I recently started to comply with the C89 standard. Declaring "i" in the middle of the function is not accepted by C89, the declaration should be moved to the beginning of the function.